src/components/hooks/useMeeting.js

```diff
@@ -22,9 +22,12 @@ export default function useMeeting(meetingID, meetingDestination) {
     const onError = (error) => {
       throw error;
     };
+    const onMeeting = (newMeeting) => {
```

Comment:
Can you add a comment as to why we're doing this?
```suggestion
    // React won't recognize the meeting attributes have been updated
    // since the state is the meeting object itself. We need to create a new
    // meeting object trigger the state change
    const onMeeting = (newMeeting) => {
```